src/basic/parse-util.h

```diff
@@ -11,6 +11,28 @@
 
 typedef unsigned long loadavg_t;
 
+#define TPM2_PCRS_MAX 24U
+
+typedef enum PcrIndex {
+        PCR_CORE_FIRMWARE      = 0,
+        PCR_PLATFORM_CONFIG    = 1,
+        PCR_EXTERNAL_CODE      = 2,
+        PCR_EXTERNAL_FIRMWARE  = 3,
+        PCR_BOOT_LOADER        = 4,
+        PCR_BOOT_LOADER_CONFIG = 5,
+        PCR_POWER_EVENTS       = 6,
+        PCR_SECURE_BOOT_POLICY = 7,
+        PCR_KERNEL_INITRD      = 9,
```

Comment:
and then insert a comment before this line, something like this:

```
/* The following names for PCRs 9…15 are based on the "Linux TPM PCR Registry"
(https://uapi-group.org/specifications/specs/linux_tpm_pcr_registry/) */
```
or so?

Reply:
> and then insert a comment before this line, something like this:
> 
> ```
> /* The following names for PCRs 9…15 are based on the "Linux TPM PCR Registry"
> (https://uapi-group.org/specifications/specs/linux_tpm_pcr_registry/) */
> ```
> 
> or so?

certainly sir.